docs/synapse/userguides/storm_ref_pivot.rstorm

```diff
@@ -7,1027 +7,1721 @@
+  and the Optic UI's `Data Model Explorer`_ list edges used by The Vertex Project by convention and which are
+  recommended for use (note that `Power-Ups`_ may create these edges).
+  
+  The Storm :ref:`storm-model`, :ref:`storm-edges`, and :ref:`storm-lift-byverb` commands can be used to work with
+  light edges in Synapse.
 
-Pivot from Tags
----------------
 
-Pivot from tags syntax allows you to pivot from a set of ``syn:tag`` nodes to the set of nodes that have those tags.
+Each traversal operation requires:
 
-**Syntax:**
+- the source node(s) for the operation;
+- the traversal operator; and
+- the target forms for the traversal.
 
-*<syn:tag>* **->** ***** | *<form>*
+Traversal is also referred to as "walking" - that is, you can traverse a light edge or walk a light edge.
 
-**Examples:**
+The traversal operator is an "arrow" with embedded parentheses. The parentheses enclose a string (commonly a
+verb) for the relationship(s) represented by the edge:
 
-- Pivot to all domains tagged with tags from any of the inbound ``syn:tag`` nodes:
+- ``-(<verb>)>``
 
+``<verb>`` can be a single edge, a list of edges, or a wildcard.
 
-.. storm-pre:: syn:tag -> inet:fqdn
-::
+Unlike property-to-property relationships, edge relationships have a **direction**. There is a "source" node
+(``n1``) and a "target" node (``n2``) for the relationship itself; an article (``media:news`` node, the ``n1``)
+can refeference an indicator such as a hash (``hash:md5`` node, the ``n2``) but it does not make sense for a
+hash to "reference" an article.
 
-    <syn:tag> -> inet:fqdn
+The pivot operator ( ``->`` ) and its variations "point" from left to right (other than a few specialized cases)
+by convention. In contrast, the traversal operator can "point" in either direction, depending on which nodes
+(which "side" of the edge relationship) are inbound. Both of the syntaxes below are equally valid:
 
+``<media:news nodes> -(refs)> <hash:md5 nodes>``
 
-- Pivot to **all** nodes tagged with tags from any of the inbound ``syn:tag`` nodes:
+``<hash:md5 nodes> <(refs)- <media:news nodes>``
 
+.. TIP::
+  
+  In Synapse, lightweight edge relationships are **explicit** relationships. The light edges used to
+  connect nodes must be defined within Synapse, and nodes joined by light edges must be explicitly linked.
 
-.. storm-pre:: syn:tag -> *
-::
+Unless otherwise specified, the target ( *<target>* ) of an edge traversal can be:
 
-    <syn:tag> -> *
+- a form name (e.g., ``hash:md5`` );
+- a partial form name (wildcard match, e.g., ``hash:*``);
+- an interface name (e.g., ``it:host:activity``);
+- a list of form names (e.g., ``( hash:sha256, file:bytes )``); or
+- a wildcard / asterisk ( ``*`` ).
 
+.. NOTE::
+  
+  You cannot specify property **values** in traversal operations. For example, the following is invalid:
+  
+  ``media:news -(refs)> inet:fqdn=woot.com``
```

Comment:
We do actually allow a comparator on walk targets, you can do things like:
```
> inet:fqdn=foo.com -(refs)> inet:fqdn^=bar
inet:fqdn=bar.com
	.created = 2021/01/29 15:39:20.398
```

Reply:
There's a story in the backlog for documenting that, SYN-2427

Reply:
Also just FYI, you can currently target secondary props and use cmprs on them in walks, but that is being removed in 3.0.

Reply:
Updated